# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
string world_frame
string robot_frame
geometry_msgs/Pose2D[] trajectory

#optional
float32 linear_velocity
float32 angular_velocity
float32 angular_error
float32 linear_error
float32 duration_error

